### ISSUE TITLE
Investigate different method to normalise mask

### ISSUE THREAD

**Issue by ADPriceless** [ADPriceless/TDA_FinalProject_Solar#11]
Article https://pytorch.org/vision/stable/auto_examples/others/plot_visualization_utils.html#semantic-seg-output states:
`normalized_masks = torch.nn.functional.softmax(output, dim=1)`

- Could this be used to normalise the masks for this project?

**Comment by ADPriceless** [comment]
From reading further, I doubt it
